src/geometries/Cube.js

```diff
@@ -124,6 +124,8 @@ function CubeTile(face, x, y, z, geometry) {
   this.z = z;
   this._geometry = geometry;
   this._level = geometry.levelList[z];
+  // Pre compute last tile size, when all is square, it's 0
+  this._tileResidue = this._level.width() % this._level.tileWidth();
```

Comment:
You also need to change the `CubeTile#neighbors` method. Currently, the logic assumes that an edge tile never meets more than one tile in the other face; this is no longer true with residual tiles.

Consider, for example, the non-residual tile at the top left of the down face. This tile meets two different tiles in the left face: the residual tile at the bottom right of the left face, and the non-residual tile immediately to the left.

A similar situation happens with some of the other corners of the cube. A strategy that might work is to have a lookup table indicating whether a particular corner is "special" or not ("special" meaning that a residual tile meets a non-residual one at that corner). Then add the additional tile to the neighbor set in the cases where the corner is "special".